scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -55,27 +80,54 @@ def get_deflection_angle(coordinates):
     return phi
 
 
-def get_sem_rotation(ccd):
+def get_sem_rotation(ccd, auto=True, channel=1):
     """
-
-    ccd: a camera object
-    return:
-    phi: rotation that needs to be added to the scan rotation in the SEM UI.
+    Find the angle of the EBeam-Deflector-x relative to the diagnostic camera.
+    If set to manual, the an x-shift must be applied to the electron beam by
+    adjusting the knobs on the AWG. When done automatically the AWG is
+    controlled by the computer and the x-shift is applied automatically.
+
+    Parameters
+    ----------
+    ccd: (odemis.model.DigitalCamera)
+        A camera object of the diagnostic camera.
+    auto: (bool)
+        True if alignment should be done automatically.
+    channel: (int)
+        Channel number of waveform generator.
+
+    Returns
+    -------
+    phi: (float)
+        The angle of the EBeam-Deflector-x relative to the diagnostic camera.
     """
     image = ccd.data.get(asap=False)
     n_spots = (8, 8)
-    spot_coordinates, translation, scaling, rotation = FindGridSpots(image, n_spots)
+    spot_coordinates, translation, scaling, rotation = FindGridSpots(image,
+                                                                     n_spots)
```

Comment:
I'd argue that this change doesn't improve readability. In general, I think that the maximum line length rule should not be applied strictly.